addon/-private/system/store.js

```diff
@@ -2034,7 +2034,14 @@ Store = Service.extend({
     // container = < 1.11
     let owner = getOwner(this);
 
-    let mixin = owner._lookupFactory('mixin:' + normalizedModelName);
+    let mixin;
+
+    if (owner.factoryFor) {
+      mixin = owner.factoryFor(`mixin:${normalizedModelName}`) && owner.factoryFor(`mixin:${normalizedModelName}`).class;
```

Comment:
Seems odd to go through the `factoryFor` twice like this, I'd suggest a refactor to:

```js
let MaybeMixin = owner.factoryFor(`mixin:${normalizedModelName}`);
mixin = MaybeMixin && MaybeMixin.class;
```